### PR TITLE
Molecule v2

### PR DESCRIPTION
**Pull request by sbesson**
This PR makes the necessary changes to test this role with Ansible 2.4 and Molecule 2.

- d29afab  creates the Molecule 2 initial setup using `molecule init scenario -r $(basename $PWD) -s default -d docker`
- 6e398ee  fixes the `yamllint` warnings and errors reported by `molecule test`
- ae81457 and a5ab22b ccopy the v1 tests and set up the molecule configuration files. 12856f2 contains the necessary changes to upgrades the tests to be compliant with Molecule 2/testinfra 1.14 including the usage of the new `host` fixture.

With https://github.com/openmicroscopy/ome-ansible-molecule-dependencies/pull/10, this PR should be tested by Travis against Ansible 2.4 in the morning https://travis-ci.org/snoopycrimecop/ansible-roles build.

Proposed tag: `3.0.2`

### CODE REVIEW

File: molecule/default/tests/test_default.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import os
```
Comment (manics) [inline molecule/default/tests/test_default.py]: This is one of the default generated test files, it doesn't contain any useful tests.

**Review by manics — COMMENTED** [review]
Looks fine apart from the extraneous test

File: molecule/default/tests/test_default.py (hunk 1 above)
Comment (sbesson) [reply molecule/default/tests/test_default.py]: Addressed by ef675e1358ebc52d039d46a19f96efa7bae42b7e

**Review by manics — APPROVED** [review]

**Comment by sbesson** [comment]
The role import onto Galaxy failed with

```
Starting import: task_id=327355, repository=openmicroscopy/ansible-role-postgresql

Task "327355" failed: 'int' object has no attribute 'get'
```

Unclear whether this is an issue with the role or upstream.

**Comment by sbesson** [comment]
After fixing `meta/main.yml` which was causing the failed import and retagging 3.0.2, this is now deployed to https://galaxy.ansible.com/openmicroscopy/postgresql.

**Pull request merged**